relations/src/r1cs/constraint_system.rs

```diff
@@ -311,12 +311,11 @@ impl<F: Field> ConstraintSystem<F> {
 
         let used_times = self.lc_num_times_used(true);
```

Comment:
Can a comment be added here as to why sinks are being counted? Its not clear to me atm.

(Also the word sink is non-intuitive to me in this context, since we're not talking about a state machine / probabilistic process of any sort)

(We can make this into a new issue / PR though)